pkg/river/internal/stdlib/stdlib.go

```diff
@@ -116,4 +118,15 @@ var Identifiers = map[string]interface{}{
 
 		return args[len(args)-1], nil
 	}),
+
+	"format":      fmt.Sprintf,
+	"join":        strings.Join,
+	"lower":       strings.ToLower,
```

Comment:
Hm, I wonder if we should do something similar to Go here and call this to_lower/to_upper. 

WDYT @grafana/grafana-agent-core-maintainers?

Reply:
I wonder if we should do `string_FUNC_NAME` though not in love with that naming? See above comment about `join` vs `concat` being similiar sounding. For instance lower/upper and numeric functions like floor and ceiling. Though to address Roberts feedback I think `to_lower` is better.

Reply:
I'm also fine with `to_lower` and `to_upper`. This is an agreement?

Reply:
Yeah, sounds like you have rough consensus on renaming the functions to to_lower/to_upper 👍

Reply:
Done 👍